prow/job/sippy.py

```diff
@@ -228,3 +235,71 @@ def last2weeks(self):
     def last4weeks(self):
         last4weeks = self._now - timedelta(days=28)
         return last4weeks.strftime(self.starttime_format)
+
+
+class DataAnalyzer():
+
+    def __init__(self, payload: dict):
+        if payload:
+            self._data_set = payload
+        else:
+            raise ValueError("payload is empty")
+
+    def is_component_readiness_status_green(self):
```

Comment:
@rioliu-rh I still think only checking status > -2 is enough. no need to check regressed_tests to calculate and determine it.

Reply:
Done